### PR TITLE
1234-abc - Add file structure

### PR DESCRIPTION
**Pull request by lasharela**
new file structure.

### CODE REVIEW

File: src/utilities/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export {};
```
Comment (Nap1400) [inline src/utilities/index.js:1]: This is a test.

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// Here we'll have component exports
+export {};
```
Comment (Dulaya) [inline src/components/index.js:2]: Looks good to me.

File: src/components/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Here we'll have component exports
```
Comment (carizzuti) [inline src/components/index.js:1]: Comment

Comment (raynesmike) [inline src/components/index.js:1]: I approve

Comment (mshago) [inline src/components/index.js:1]: comment

File: src/components/index.js [hunk 4]
```diff
@@ -1 +1,2 @@
...
 export {};
```
Comment (leo-he-1) [inline src/components/index.js:2]: Cool Change

File: src/components/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export {};
```
Comment (Josetf2396) [inline src/components/index.js:2]: -Que?

**Comment by aahenry51** [comment]
Hello

File: src/components/index.js [hunk 6]
```diff
@@ -1 +1,2 @@
+// Here we'll have component exports
```
Comment (rahmiunalan) [inline src/components/index.js:1]: so is this what i need to do

File: src/utilities/index.js (hunk 1 above)
Comment (lasharela) [reply src/utilities/index.js:1]: @rahmiunalan do you know why it's here?

**Review by raynesmike — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by mshago — APPROVED** [review]

**Review by carizzuti — APPROVED** [review]

**Review by leo-he-1 — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by Nap1400 — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by rahmiunalan — COMMENTED** [review]
ok

**Pull request merged**